lib/bugsnag/api/client/errors.rb

```diff
@@ -8,23 +8,23 @@ class Client
       module Errors
         # List the Errors on a Project
         #
+        # @argument id [String] optional ID of error to retrieve
+        #
         # @option base [String] Only Error Events occuring before this time will be returned
         # @option sort [String] Which field to sort by, one of: last_seen, first_seen, users, events, unsorted
         # @option direction [String] Which direction to sort the result by, one of: asc, desc
         # @option filters [Filters] An optional filters object, see http://docs.bugsnagapiv2.apiary.io/#introduction/filtering
         # @return [Array<Sawyer::Resource>] List of Project Errors
         # @see http://docs.bugsnagapiv2.apiary.io/#reference/errors/errors/list-the-errors-on-a-project
         def errors(project_id, id=nil, options = {})
-          paginate "projects/#{project_id}/errors", options
+          if id.nil?
+            paginate "projects/#{project_id}/errors", options
+          else
+            get "projects/#{project_id}/errors/#{id}", options
+          end
         end
 
-        # View an Error
-        #
-        # @return [Sawyer::Resource] Requested Error
-        # @see http://docs.bugsnagapiv2.apiary.io/#reference/errors/errors/view-an-error
-        def error(project_id, id, options = {})
-          get "projects/#{project_id}/errors/#{id}", options
-        end
+        alias error errors
```

Comment:
Will Ruby show the `Errors` Docs for this alias as well?

Reply:
We don't have RDoc documentation being generated for this repo atm (although it'd be nice to have).  Looks like RDoc does deal with linking aliased methods to their parents as well, so this shouldn't be an issue.